examples/chaincode/go/asset_management_with_roles/asset_management_with_roles.go

```diff
@@ -17,10 +17,12 @@ limitations under the License.
 package main
```

Comment:
Since this chaincode is a sample, it is worth explaining what is being done, especially the strategy to leverage the attributes for asset assignment and transferring -- why should we use attributes?

The comment block about the chaincode doesn't seem to go with this chaincode, but it looks like a left over from the previous copy. The referenced URL is also confusing.